README.md

```diff
@@ -23,9 +25,30 @@ And then execute:
 
     $ bundle
 
-Or install it yourself as:
+### Option 2: Install local copy
```

Comment:
I agree the current docs aren't great. I should try this out before committing it though...

Reply:
Also I can just put the gemspec changes in and save the readme edits for later - let me know what you think!

Reply:
Yeah sorry, could you split the readme out and I will commit this.

Reply:
Done!